webapp/components/needs_team.jsx

```diff
@@ -103,6 +106,11 @@ export default class NeedsTeam extends React.Component {
         });
 
         Utils.applyTheme(this.state.theme);
+
+        if (UserAgent.isIosSafari()) {
```

Comment:
Is there some particular reason you chose NeedsTeam? Doesn't this effect everything and should be on Root?